README.md

```diff
@@ -36,581 +34,25 @@ The following set of features makes Artipie unique among all others:
-
-```bash
-$ nuget install MyLib -Version 1.0.0 -Source=http://localhost:8080/my-nuget/index.json
-```
-
-### Gem Repo
-
-Try this `gem.yaml` file:
-
-```yaml
-repo:
-  type: gem
-  storage:
-    type: fs
-    path: /tmp/artipie/data/my-nuget
-```
-
-Publish a gem:
-
-```bash
-$ gem push my_first_gem-0.0.0.gem --host http://localhost:8080/gem
-```
-
-Install a gem:
-
-```bash
-$ gem install my_first_gem --source http://localhost:8080/gem
-```
-
-### Helm chart repo
-
-Try this `helm.yaml` file:
-
-```yaml
-repo:
-  type: helm
-  storage:
-    type: fs
-    path: /tmp/artipie/data/helm-charts
-```
-
-Publish a chart:
-
-```bash
-$ curl --data-binary "@my_chart-1.6.4.tgz" http://localhost:8080/helm
-```
-
-Install a chart:
-
-```bash
-$ helm repo add artipie http://localhost:8080/helm/charts
-$ helm install my_chart artipie
-```
-
-### Docker Repo
-
-Try this `docker.yaml` file:
-
-```yaml
-repo:
-  type: docker
-  storage:
-    type: fs
-    path: /tmp/artipie/data/my-docker
-```
-
-Docker registry has to be protected by HTTPS and should have no prefix in path.
-In order to access this Docker repository it is required to run a reverse proxy such as
-[nginx](https://nginx.org/) or [lighttpd](https://www.lighttpd.net/) to protect Artipie
-with HTTPS and add forwarding of requests from `my-docker.my-company.com/<path>` to
-`my-artipie.my-company.com/my-docker/<path>`.
-Then to push your Docker image use the following command:
-
-```bash
-$ docker push my-docker.my-company.com/my-image
-```
-
-To pull the image use the following command:
-
-```bash
-$ docker pull my-docker.my-company.com/my-image
-```
-
-### Docker Proxy Repo
-
-Try this `docker-proxy.yaml` file to host a proxy to `mcr.microsoft.com` registry:
-
-```yaml
-repo:
-  type: docker-proxy
-  settings:
-    host: mcr.microsoft.com
-```
-
-Artipie will redirect all pull requests to specified registry.
-
-Proxy repository supports caching in local storage.
-To enable it and make previously accessed images available when source repository is down 
-add `storage` section to config:
-
-```yaml
-repo:
-  type: docker-proxy
-  settings:
-    host: mcr.microsoft.com
-  storage:
-    type: fs
-    path: /tmp/artipie/data/my-docker-cache
-```
-
-### Python repo
-
-Try this `pypi.yaml` file:
-
-```yaml
-repo:
-  type: pypi
-  storage:
-    type: fs
-    path: /tmp/artipie/data/python-repo
-```
-
-Publish a package(whl or tar.gz):
-  * Install twine utility, if you don't do it already [docs](https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives).
-```bash
-$ python3 -m pip install --user --upgrade twine
-```
-  * build the package, as described in python docs
-  * upload to server with a command
-```bash
-$ python3 -m twine upload --repository-url http://localhost:8080/pypi/ -u user.name -p pass testpkg/dist/*
-```
-
-Install a package:
-
-```bash
-$ pip install --index-url http://localhost:8080/pypi/ testpkg
-```
-
-## Additional configuration
```

Comment:
@Sammers21 what about "Additional configuration", "Multitenancy" and "Metrics" sections? I think we need some separate space for them too. Maybe for now we can simply copy all thin information to another file, add corresponding links and todo to refactor this file.

Reply:
@olenagerasimova, I think we should have a documentation section for that. The readme has a reference to documentation.